content/01.main_content.md

```diff
@@ -24,10 +26,19 @@ Proxy predictions of ethnicity via name origin are more difficult still; choosin
   For instance, classifiers are usually unable to distinguish if names of Hispanic origin come from the Iberian Peninsula or from Latin America [@doi:10.1016/j.cels.2021.07.007].
   If a target and background population had identical probabilistic proportions of Hispanic names, but the target group primarily consisted of Iberian individuals, underrepresentation of Latin American scientists would go unnoticed in an exclusively name-based analysis.
 
+Beyond accurate classification of individuals' social identifiers, auditing systems also require a comparable background distribution.
+  For example, in Le et al. [@doi:10.1016/j.cels.2021.07.007], in order to identify disparities in ISCB honorees, they compared against scientists' publication records.
+  While seemingly straightforward, there exist many choices on how to calculate publication rates. 
+  The authors had to decide which journals would be included in their publication record, whether the publications should be weighted by citation count, and if authorship position should be considered.
+  Choosing the correct reference distribution will be specific to the application, but the selection criteria of the background distribution should be annotated and justified.
```

Comment:
I'd switch this to "The most appropriate reference distribution..."

Reply:
ah, yes, good point
https://github.com/greenelab/disparities_opinion/pull/16/commits/2021d8537658d4db25727b3608e952ecddaae4bc